.github/workflows/builders.yaml

```diff
@@ -115,3 +115,21 @@ jobs:
           isolated-network: 1
           ovnk-networking: 1
           build: bootc-image
+
+  multi-node-bootc:
```

Comment:
With the suggestion to have test-cluster-nodes argument to the build action, maybe we do not need a special builder here? We could just add more nodes to the existing jobs?

Reply:
As discussed, let's pick a couple of existing tests from the list. 
Your choice 👍